src/handlers/pinata.ts

```diff
@@ -0,0 +1,53 @@
+import {
+  // @ts-expect-error ts-migrate(2305) FIXME: Module '"react-native-dotenv"' has no exported mem... Remove this comment to see the full error message
+  PINATA_API_KEY,
+  // @ts-expect-error ts-migrate(2305) FIXME: Module '"react-native-dotenv"' has no exported mem... Remove this comment to see the full error message
+  PINATA_API_SECRET,
+  // @ts-expect-error ts-migrate(2305) FIXME: Module '"react-native-dotenv"' has no exported mem... Remove this comment to see the full error message
+  PINATA_API_URL,
+  // @ts-expect-error ts-migrate(2305) FIXME: Module '"react-native-dotenv"' has no exported mem... Remove this comment to see the full error message
+  PINATA_GATEWAY_URL,
+} from 'react-native-dotenv';
+import RNFS from 'react-native-fs';
+
+type UploadFilesResponse = {
+  IpfsHash: string;
+  PinSize: number;
+  Timestamp: string;
+  isDuplicate: boolean;
+};
+export type UploadImageReturnData = UploadFilesResponse & {
+  url: string;
+};
+
+export async function uploadImage({
```

Comment:
just want to check are we handling uploadImage failures appropriately / in an expected manner?

Reply:
addressed by @jxom here https://github.com/rainbow-me/rainbow/pull/3173